NuCivic/dkan
# Add test criteria for resource deletion in harvester

Issue: https://github.com/NuCivic/dkan/issues/1969

## Description

Adds test criteria for resources count, groups info, and deleted resources.

## User story

As a site manager I want to see the number of resources that will be deleted in the confirmation message, along with information about groups when confirming deletion of a harvester source so I can be properly informed about what is and what is not going to happen.

## How to reproduce

1. Delete a Harvester source and see the confirmation message.
2. See that resources associated with source datasets have been deleted.

## QA Steps

- [x] Add a Harvest source if one does not exist (https://project-open-data.cio.gov/v1.1/examples/catalog-sample-extended.json for example)
- [x] Delete one or more Harest sources
- [x] See that the confirmation message says: 
```
The following sources were selected to be deleted:
Harvest Source Name: 18 Datasets, 29 Resources
No groups generated during harvest will be affected by this action. They must be manually managed.
```
- [x] See that resources are deleted if datasets are deleted

## Reminders
